python/ray/external_storage.py

```diff
@@ -46,6 +49,13 @@ def restore_spilled_objects(self, keys: List[bytes]):
             keys: A list of bytes corresponding to the spilled objects.
         """
 
+    def validate(self):
+        """Check if the external storage configuration is valid.
```

Comment:
Can we do this in the constructor instead?

Reply:
Ah yeah. That's a good idea. Will fix.